App/src/Command/HandleBalanceCommand.php

```diff
@@ -25,6 +32,70 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $io = new SymfonyStyle($input, $output);
+        $scenario = $io->choice(
+            'Selectionner le scénario à executer:',
+            [
+                "First",
+                "Second",
+                "Third",
+                "Fourth"
+            ]
+        );
+
+        $datas = match ($scenario) {
+            'First' => [
+                new Expense(9 * 100, "Alice", ["Alice", "Charles", "Camille"], "Bouteille  d'eau"),
+                new Expense(6 * 100, "Charles", ["Charles"], "Sandwich"),
+                new Expense(12 * 100, "Charles", ["Alice", "Camille"], "Nourriture"),
+                new Expense(36 * 100, "Camille", ["Alice", "Charles", "Camille"], "Essence")
+            ],
+            'Second' =>
+            [
+                new Expense(10 * 100, "Pierre", ["David", "Emilie", "Florence"], "Taxi")
+            ],
+            'Third' =>
+            [
+                new Expense(10 * 100, "George", ["George", "Helene"], "Petit dèj"),
+                new Expense(15 * 100, "Helene", ["George"], "Déjeuner"),
+                new Expense(20 * 100, "George", ["Helene"], "Diner"),
+            ],
+            'Fourth' =>
+            [
+                new Expense(50 * 100, "Isabelle", ["Isabelle", "Julien", "Leo"], "Peinture"),
+                new Expense(50 * 100, "Julien", ["Isabelle", "Julien", "Leo"], "Faux gazon"),
+                new Expense(50 * 100, "Leo", ["Isabelle", "Julien", "Leo"], "Plomb"),
+            ],
+        };
+
+        $this->entityManager->beginTransaction();
+        foreach ($datas as $expense) {
+            $this->entityManager->persist($expense);
+        }
+        $this->entityManager->flush();
+
+        $this->showBalance($output);
+
+        $this->entityManager->rollback();
+
         return Command::SUCCESS;
     }
+
+    private function showBalance(OutputInterface $output)
+    {
+
+        $expenses = $this->expenseRepository->findAll();
```

Comment:
if I run multiple scenario, if we recover all the expenses, will we recover the balance sheet for previous scenarios? maybe it's out of scope for now ?

Reply:
For now, I push the expenses of one and only group in the BDD, I didn't managed yet the groups entity so for now it's in the scope :) The next step is to render in twig the different groups and their balance so stay tuned for the next PR !!!!!